test/unit/option_spec.lua

```diff
@@ -0,0 +1,62 @@
+
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+local check_ff_value = function(ff)
+  return option.check_ff_value(to_cstr(ff))
+end
+
+local get_sts_value = function()
+  return option.get_sts_value()
+end
+
+describe('check_ff_value string', function()
```

Comment:
what is the purpose of "string" in the description? only the function name `check_ff_value` is needed